forward_engineering/api.js

```diff
@@ -232,4 +232,4 @@ module.exports = {
 
 const getScript = (structure) => {
```

Comment:
api.js 's function should invoke alterScriptBuilder's functions for building the alter script and checking if it contains 'DROP' statements. I do not see these changes